### PR TITLE
Ids-multipart: Replace usage of TransferProcessStore with -Manager

### LINKED ISSUE #301: ArtifactRequestHandler should not access the TPS
ArtifactRequestHandler should not call the TransferProcessStore to initiate a new transfer process. Instead, it should use the TransferProcessManager.

### PR DESCRIPTION
**Pull request by bcronin90**
Replaces usage of TransferProcessStore with TransferProcessManager in ids-multipart.

Closes #301 

Brendan Cronin brendan.cronin@daimler.com, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)

### CODE REVIEW

**Review by paullatzelsperger — APPROVED** [review]
LGTM

File: data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/IdsMultipartApiServiceExtension.java [hunk 1]
```diff
@@ -144,10 +145,10 @@ private void registerControllers(ServiceExtensionContext serviceExtensionContext
         List<Handler> handlers = new LinkedList<>();
         handlers.add(descriptionHandler);
 
-        TransferProcessStore transferProcessStore = serviceExtensionContext.getService(TransferProcessStore.class);
+        TransferProcessManager transferProcessManager = serviceExtensionContext.getService(TransferProcessManager.class);
```
Comment (DominikPinsel) [inline data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/IdsMultipartApiServiceExtension.java:148]: Maybe you can update the required set in `IdsMultipartApiServiceExtension`, too. There is still the `TransferProcessStore` in it.

```java
@Override
    public Set<String> requires() {
        return Set.of(IdentityService.FEATURE,
                "dataspaceconnector:transferprocessstore",
                "edc:ids:core",
                "edc:ids:transform:v1");
    }
```

**Review by jimmarino — APPROVED** [review]
LGTM, please just remove the requires mentioned by @DominikPinsel

**Comment by bcronin90** [comment]
> LGTM, please just remove the requires mentioned by @DominikPinsel

@jimmarino I don't really understand this. I replaced the entry with `"dataspaceconnector:transfer-process-manager"`. Would it be better to remove it entirely?

**Comment by jimmarino** [comment]
> > LGTM, please just remove the requires mentioned by @DominikPinsel
> 
> @jimmarino I don't really understand this. I replaced the entry with `"dataspaceconnector:transfer-process-manager"`. Would it be better to remove it entirely?

@bcronin90 that's fine, I hadn't seen you already incorporated Dominik's suggestion.

**Pull request merged**